target/linux/ath79/nand/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -25,6 +25,17 @@ case "$FIRMWARE" in
 		;;
 	esac
 	;;
+"ath10k/pre-cal-pci-0000:00:00.0.bin")
+	case $board in
+	glinet,gl-x1200-nor|\
+	glinet,gl-x1200-nor-nand)
+		caldata_extract "art" 0x5000 0x2f20
```

Comment:
Please define the position of the cal data in device tree. These commits are doing it for a different device: d4b3b23942336e22ef36ccbcf584efa6a55cc408 and 2d3321619b2b0a1eaabec3f47353e4e0aee3dc31.

Reply:
hi hauke, I have modified according to your suggestion, thank you for your help